src/test/lambda/commands/deploySamApplication.test.ts

```diff
@@ -0,0 +1,346 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2018 -> 2019 ; only do this for new files for now (dashes, commas, and spaces aren't supported until 557 merges)